narwhal/storage/src/certificate_store.rs

```diff
@@ -270,8 +270,8 @@ impl CertificateStore {
             .collect()
     }
 
-    /// Retrieves the certificates of the last round
-    pub fn last_round(&self) -> StoreResult<Vec<Certificate>> {
+    /// Retrieves the certificates of the last round and the round before that
+    pub fn last_two_rounds_certs(&self) -> StoreResult<Vec<Certificate>> {
```

Comment:
Ok since we are doing this maybe it would worth making it a bit more generic? For example like 

```
/// Retrieves the certificates of the last `num_of_rounds`  rounds starting from the end
/// The result is returned ordered from the lowest --> highest round. If no more than
/// num_of_rounds exist then all the available ones will be fetched.
///
/// Example:  
/// * last_rounds(1) , will fetch the certificates of the last round only
/// * last_rounds(2) , will fetch the certificates of the last and previous round
///
last_rounds(num_of_rounds: u64) -> StoreResult<Vec<Certificate>>
```